src/main/java/org/spongepowered/api/service/event/EventManager.java

```diff
@@ -34,13 +35,23 @@
     /**
      * Registers an object to receive {@link Event}s.
      *
-     * @param plugin A plugin instance
+     * @param plugin The plugin instance
      * @param obj The object
      * @throws IllegalArgumentException Thrown if {@code plugin} is not a plugin instance
      */
     void register(Object plugin, Object obj);
```

Comment:
(Why not?) Are you adding some clarifications to the javadocs? Or is it meant to be multi-functional.

Because this is an established method and everyone knows about it at this point.

    It would be quite useful to be able to realign the priority of the event handlers to better match another plugin's handler's priority. Example execute the Login plugin's handler after the daily join gift handler.

Way to easy for abuse. Plugins shouldn't have any control over the event pipeline.